### DISCUSSION TITLE
Check waitinglist when seats become available

### DISCUSSION THREAD

**Discussion by roysinn** [liedekef/events-made-easy#318] [General]
I'm using the "Check waitinglist when seats become available" feature for an event and have increased that event's "Seats" to approve waitlisted registrations. I assumed that just increasing the Seats for the event would trigger that action but it doesn't seem to. Is this how EME functions? A cancellation does, however, trigger the process so I registered and cancelled and that set the wheel turning. Are there other actions that will process a waitinglist?

**Comment by liedekef** [comment]
Indeed, currently adding seats doesn't trigger the recalc of currently waiting list seats. That would need to be coded up (is a use-case I didn't think of)

**Reply by liedekef** [reply]
The next version will fix this. The changeset: 
https://github.com/liedekef/events-made-easy/commit/41fa6825e6fcb243b3784722d97426d0bcb9fda6

**Comment by roysinn** [comment]
Great. Thanks Frankie. In the meantime I have a workaround.

**Reply by liedekef** [reply]
it is already in the released version :-)

**Reply by roysinn** [reply]
Many thanks, Franky.
